aetherpy/io/read_routines.py

```diff
@@ -287,6 +287,70 @@ def read_aether_ascii_header(filename):
     return header
 
 
+#  Gets dimensions, variables, attributes (may not have attributes)
+# --will require work to integrate with aetherpy
+# --not same format as GITM file headers
+# --only meant for reading block-based Aether netcdf files
+def read_blocked_netcdf_header(filename):
+    # Checks for file existence
+    if not os.path.isfile(filename):
+        raise IOError('unknown aether netCDF file: {:}'.format(filename))
+
+    header = {'filename': filename}  # Included for compatibility
+
+    with Dataset(filename, 'r') as ncfile:
+        # Process header information: nlons, nlats, nalts, nblocks
+        header['nlons'] = len(ncfile.dimensions['lon'])
+        header['nlats'] = len(ncfile.dimensions['lat'])
+        header['nalts'] = len(ncfile.dimensions['z'])
+        header['nblocks'] = len(ncfile.dimensions['block'])
+
+        # Included for compatibility ('vars' slices time out for some reason)
+        header['vars'] = list(ncfile.variables.keys())[1:]
+        header['time'] = epoch_to_datetime(
+            np.array(ncfile.variables['time'])[0])
+
+    return header
+
+
+def read_blocked_netcdf_file(filename, file_vars=None):
+    # Checks for file existence
+    if not os.path.isfile(filename):
+        raise IOError('unknown aether netCDF file: {:}'.format(filename))
+
+    # NOTE: Includes header information for easy access until
+    #       updated package structure is confirmed
+    # Initialize data dict with defaults (will remove these defaults later)
+    data = {'filename': filename,
+            'units': '',
+            'long_name': None}
+
+    with Dataset(filename, 'r') as ncfile:
+        # Process header information: nlons, nlats, nalts, nblocks
+        data['nlons'] = len(ncfile.dimensions['lon'])
+        data['nlats'] = len(ncfile.dimensions['lat'])
+        data['nalts'] = len(ncfile.dimensions['z'])
+        data['nblocks'] = len(ncfile.dimensions['block'])
+
+        # Included for compatibility
+        data['vars'] = [var for var in ncfile.variables.keys()
+                        if file_vars is None or var in file_vars]
+
+        # Fetch requested variable data
+        for var in data['vars']:
+            data[var] = np.array(ncfile.variables[var])  # key is var name
+
+        # TODO: Fetch all attribute data
```

Comment:
```suggestion
        # TODO(#issue number): Fetch all attribute data
```

Make sure there is an issue for this and put the issue number here.